compiler/src/dotty/tools/dotc/reporting/diagnostic/ErrorMessageID.java

```diff
@@ -91,6 +91,7 @@
     AnonymousFunctionMissingParamTypeID,
     SuperCallsNotAllowedInlineID,
     ModifiersNotAllowedID,
+    WildcardOnTypeArgumentNotAllowedOnNewID
```

Comment:
This line should end in `,` so that the diff is single lines :)

(omg, java has trailing commas! :fire: :fire: 🔥 )